wikipron/languagecodes.py

```diff
@@ -252,10 +252,88 @@
     "tetelcingo nahuatl": "Tetelcingo Nahuatl",
     # Bouyei. ISO 639-3 only.
     "pcc": "Bouyei",
+    "bouyei": "Bouyei",
     # Lamboya. ISO 639-3 only.
-    "lmy": "Lamboya",
+    "lmy": "Laboya",
+    "laboya": "Laboya",
     # Moroccan Arabic. ISO 639-3 only.
     "ary": "Moroccan Arabic",
+    "moroccan arabic": "Moroccan Arabic",
     # Mandarin Chinese. ISO 639-3 only.
-    "cmn": "Chinese"
+    "cmn": "Chinese",
+    "chinese": "Chinese",
+    # Abkhaz. Would be Abkhazian in ISO 639.
+    "abk": "Abkhaz",
+    "abkhaz": "Abkhaz",
+    # Avar. Would be Avaric in ISO 639.
+    "ava": "Avar",
+    "avar": "Avar",
+    # Buryat. Would be Buriat in ISO 639.
+    "bua": "Buryat",
+    "buryat": "Buryat",
+    # Chukchi. Would be Chukot in ISO 639.
+    "ckt": "Chukchi",
+    "chukchi": "Chukchi",
+    # Burushaski. ISO 639-3 only.
+    "bsk": "Burushaski",
+    "burushaski": "Burushaski",
+    # Evenki. Not in iso639 lib.
+    "evn": "Evenki",
+    "evenki": "Evenki",
+    # Southern Yukaghir. Not in iso639 lib.
+    "yux": "Southern Yukaghir",
+    "southern yukaghir": "Southern Yukaghir",
+    # Tundra Nenets. Not in iso639 lib.
+    "yrk": "Tundra Nenets",
+    "tundra nenets": "Tundra Nenets",
+    # Estonian. Not in iso639 lib.
+    "ekk": "Estonian",
+    "estonian": "Estonian",
+    # Livvi. Not in iso639 lib.
+    "olo": "Livvi",
+    "livvi": "Livvi",
+    # Kildin Sami. Not in iso639 lib.
+    "sjd": "Kildin Sami",
+    "kildin sami": "Kildin Sami",
+    # Northern Yukaghir. Not in iso639 lib.
+    "ykg": "Northern Yukaghir",
+    "northern yukaghir": "Northern Yukaghir",
+    # Nanai. Not in iso639 lib.
+    "gld": "Nanai",
+    "nanai": "Nanai",
+    # Persian. Not in iso639 lib.
+    "pes": "Persian",
+    "persian": "Persian",
```

Comment:
My spot checking tells me that some Persian entries have the dialect indication (e.g., "Iranian Persian" in [this entry](https://en.wiktionary.org/wiki/%D8%A2%D9%84%D9%88%D8%AF%DA%AF%DB%8C#Persian)), which suggests that we should use the macro-language code `per` here (and a WikiPron user who would like to target "Iranian Persian" specifically will have to specify the dialect flag for that).

To not block this PR further, I'd be fine with merging this PR as-is, and then the WikiPron developers can fix this ourselves.